examples/policies/policy-definition/README.md

```diff
@@ -34,7 +34,7 @@ The ARM template provided in this folder shows how a new policy definition is cr
 
 ## Deploy using AzOps
 
-See these [instructions]( https://github.com/azure/azops/wiki/deployments) for how to deploy ARM templates with the AzOps GitHub Actions/DevOps pipeline.
+See these [instructions](./ https://github.com/azure/azops/wiki/deployments) for how to deploy ARM templates with the AzOps GitHub Actions/DevOps pipeline.
```

Comment:
This doesn't look right? (realise I may have made this update whilst bulk fixing other things)

Reply:
Removed